src/app/app.component.spec.ts

```diff
@@ -39,4 +45,86 @@ describe('AppComponent', () => {
     fixture.detectChanges();
     expect(app).toBeTruthy();
   });
+
+  describe('test route navigate', () => {
```

Comment:
following the pattern for other unit tests, the describe title should be the name of the function you are testing, so you should have individual describe blocks for router.initialNavigation & router.navigateByUrl

Reply:
Yup, thanks for the heads up. I am going to amend it to make it more sensible